### ISSUE TITLE
Feature: Text to speech

### ISSUE THREAD

**Issue by map7** [asciinema/asciinema#574]
Instead of recording audio which takes a lot of space, what about using 'text to speech' instead and getting the browser to play the correct text at the right time of the recording.

https://dev.to/asaoluelijah/text-to-speech-in-3-lines-of-javascript-b8h

This would keep the size of the recordings down but add the ability to put speech alongside the videos. It could also double as captions.

**Comment by ku1ik** [comment]
Ooh, that's interesting idea. This would indeed handle audio in a similar fashion as we handle "video" - through minimal amount of information needed to synthesize the output. Nice!

Have you used TTS API "in anger"? Is it widely supported across browsers these days? I see the original post is from 2020 so I guess there's good chance it's already out there, at least in major browsers.

**Comment by map7** [comment]
Yeah I was thinking about your problem whilst listening to you on changelog. I haven't used TTS but thought there would be something which could do text to speech and it would fit in with your project.

**Comment by ku1ik** [comment]
Got it. Thanks for the idea, I'll explore this for sure!

**Comment by ku1ik** [comment]
Moving to asciinema-player as this is mostly player related feature.
